### ISSUE TITLE
Fix Request for pmd7 AvoidModifyingObjectMapper false positive

### ISSUE THREAD

**Issue by jborgers** [jborgers/PMD-jPinpoint-rules#400]
```
import com.fasterxml.jackson.databind.ObjectMapper;

public class Issue {

    private void setTransformOptions(ObjectMapper objectmapper, Object obj) {
  
        obj.set();
    }
}

```
